e2e-tests/playwright/support/pageObjects/page-obj.ts

```diff
@@ -20,3 +20,32 @@ export const SettingsPagePO = {
   userSettingsMenu: 'button[data-testid="user-settings-menu"]',
   signOut: 'li[data-testid="sign-out"]',
 };
+
+export const RoleFormPO = {
+  roleName: 'input[name="name"]',
+  roledescription: 'input[name="description"]',
+  addUsersAndGroups: 'input[name="add-users-and-groups"]',
+  addPermissionPolicy: 'button[name="add-permission-policy"]',
+  selectMember: (label: string) => `span[data-testid="${label}"]`,
+  selectPermissionPolicyPlugin: (row: number) =>
```

Comment:
Could we use the permission name or something similar for selection? The row number might change in the future, which could lead to selecting the wrong permission.

Reply:
Yea, I'm kinda doing the same. The row number here is the row where permission and policy are added. This will not change in the future

```
    await this.page.click(RoleFormPO.selectPermissionPolicyPlugin(0));
    await this.uiHelper.optionSelector('catalog');
    await this.page.click(RoleFormPO.selectPermissionPolicyPermission(0));
    await this.uiHelper.optionSelector('catalog-entity');
    await this.page.uncheck(RoleFormPO.selectPolicy(0, 1, 'Delete'));
```